packages/client/src/components/Cards/CardWrapper.tsx

```diff
@@ -0,0 +1,30 @@
+import { PropsWithChildren, MouseEvent, useRef } from 'react';
+import { Box } from 'theme-ui';
+import { useSpring, animated, config } from 'react-spring';
+
+type CardWrapperProps = PropsWithChildren<{ onClick: (e: MouseEvent<HTMLElement>) => void; moving: boolean }>;
+
+const INITIAL_POSITION = { y: 0, x: 0 };
+
+export const CardWrapper = ({ moving, ...rest }: CardWrapperProps) => {
+  const ref = useRef<HTMLDivElement | null>(null);
+
+  const { x, y } = ref.current?.getBoundingClientRect() ?? INITIAL_POSITION;
```

Comment:
What do the `?` mean here...?

Reply:
This is optional chaining, it's a new feature in js/ts check it out https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining